docs/content/users/providers/lagoon.md

```diff
@@ -0,0 +1,18 @@
+# Amazee Lagoon Integration
+
+DDEV provides integration with the [Amazee Lagoon](https://lagoon.sh/), allowing Lagoon users to quickly download database and files from a Lagoon project to the local DDEV project.
+
+## Amazee Lagoon Per-Project Configuration
+
+1. Check out the Lagoon project and configure it with [`ddev config`](../usage/commands.md#config). You’ll want to use [`ddev start`](../usage/commands.md#start) and make sure the basic functionality is working.
+2. Configure an SSH key at on the [Amazee Cloud Dashboard](https://dashboard.amazeeio.cloud/settings).
+3. Add LAGOON_PROJECT and LAGOON_ENVIRONMENT variables to your project in 'web_environment' or a '.ddev/.env'. For example, `ddev config --web-environment-add="LAGOON_PROJECT=<project-name> LAGOON_ENVIRONMENT=<environment-name>"`.
```

Comment:
Wrap environment variable names and filenames in backticks, clarify YAML vs. `.env` file, run commands:

```suggestion
3. Add `LAGOON_PROJECT` and `LAGOON_ENVIRONMENT` variables to your project using `'web_environment'` in its YAML configuration or a `.ddev/.env` file. For example, run `ddev config --web-environment-add="LAGOON_PROJECT=<project-name> LAGOON_ENVIRONMENT=<environment-name>"`.
```

Reply:
I wonder how this passed spellcheck.